Components/Utils/CountryCodeResolver.php

```diff
@@ -18,15 +18,29 @@
  */
 class CountryCodeResolver
 {
-    /** @var \Shopware\Components\Model\ModelManager */
+    /**
+     * @var \Shopware\Components\Model\ModelManager
```

Comment:
Only class name or full namespace? in previous files we use only class name.
Need to decide what to use in the future